services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/TransferManagerFactory.java

```diff
@@ -68,7 +64,7 @@ private static Supplier<S3AsyncClient> defaultS3AsyncClient() {
         if (crtInClasspath()) {
             return S3AsyncClient::crtCreate;
         }
-        return S3AsyncClient::create;
+        return () -> S3AsyncClient.builder().multipartEnabled(true).build();
```

Comment:
Can we update javadocs in `S3TransferManager` to specify multipart enabled cases

Reply:
done